home/.claude/CLAUDE.md

```
@@ -71,10 +73,13 @@
-  - `$HOME/work/clan/clan-core`
+- `$HOME/git/nixpkgs`
+- `$HOME/git/linux`
+- `$HOME/git/nix`
+- `$HOME/work/clan/clan-core`
 - Use Kagi instead of the Websearch tool for better search results:
-  - `kagi-search "nixpkgs buildPythonPackage examples"`
-  - `kagi-search -j "nix flake inputs follows" | jq -r '.[0].url'`
+- `kagi-search "nixpkgs buildPythonPackage examples"`
+- `kagi-search -j "nix flake inputs follows" | jq -r '.[0].url'`
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)